client/app/deskaas/deskaas-details/deskaas-details.html

```diff
@@ -1,165 +1,228 @@
+                            <oui-skeleton randomized></oui-skeleton>
+                            <oui-skeleton randomized></oui-skeleton>
+                            <oui-skeleton randomized></oui-skeleton>
+                            <oui-skeleton randomized></oui-skeleton>
+                            <oui-skeleton randomized></oui-skeleton>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <div ng-if="!($ctrl.flags.init.serviceInfos || $ctrl.flags.init.details)">
+                        <oui-tile-definition
+                            data-ng-if="$ctrl.services.offer" 
+                            data-term="{{::'common_tile_subscription_offer' | translate}}">
+                            <oui-tile-description>{{$ctrl.services.offer}}</oui-tile-description>
+                            <oui-action-menu data-compact align="end" data-ng-if="$ctrl.actions.changeOffer.isAvailable()">
+                                <oui-action-menu-item data-text="{{::$ctrl.actions.changeOffer.text}}"
+                                    data-aria-label="{{::$ctrl.actions.changeOffer.text}}"
+                                    data-on-click="$ctrl.actions.changeOffer.callback()">
+                                </oui-action-menu-item>
+                            </oui-action-menu>
+                        </oui-tile-definition>
+                        <oui-tile-definition
+                            data-term="{{($ctrl.services.renew.forced || $ctrl.services.renew.automatic) ? 'common_tile_subscription_autorenew' : 'common_tile_subscription_expiration_date' | translate}}">
+                            <oui-tile-description>{{$ctrl.services.expiration | momentFormat:'LL'}}</oui-tile-description>
+                            <oui-action-menu data-compact align="end" data-ng-if="$ctrl.actions.manageAutorenew.isAvailable()">
+                                <oui-action-menu-item data-text="{{::$ctrl.actions.manageAutorenew.text}}"
+                                    data-aria-label="{{::$ctrl.actions.manageAutorenew.text}}"
+                                    data-href="{{$ctrl.actions.manageAutorenew.href}}"
+                                    external>
+                                </oui-action-menu-item>
+                            </oui-action-menu>
+                        </oui-tile-definition>
+                        <oui-tile-definition
+                            data-term="{{::'common_tile_subscription_contact' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="$ctrl.services.contactAdmin"></span>
+                                <small data-translate="common_tile_subscription_contact_admin"></small>
+                            </oui-tile-description>
+                            <oui-tile-description>
+                                <div>
```

Comment:
Why use a `div` here and not in the item above

Reply:
there is a bug with oui-tile-description. It does not show content of each description on different line. As a work around, I used div to show content on separate line. I have created a bug (UK-46) on UI-KIT tema to track it.